blueflood-elasticsearch/pom.xml

```diff
@@ -75,7 +75,7 @@
     <dependency>
       <groupId>com.github.tlrx</groupId>
       <artifactId>elasticsearch-test</artifactId>
-      <version>1.1.0</version>
+      <version>1.2.1</version>
```

Comment:
I decided to just upgraded the elasticsearch-test lib to version 1.2.1 (latest stable) here just because it's good to use the latest stable version, but it's not strictly necessary for this fix.